pkg/netflow/common/flowtype.go

```diff
@@ -0,0 +1,72 @@
+package common
+
+import (
+	"fmt"
+)
+
+// FlowType represent the flow protocol (netflow5,netflow9,ipfix, sflow, etc)
+type FlowType string
+
+// Flow Types
+const (
+	TypeIPFIX    FlowType = "ipfix"
+	TypeSFlow5   FlowType = "sflow5"
+	TypeNetFlow5 FlowType = "netflow5"
+	TypeNetFlow9 FlowType = "netflow9"
+	TypeUnknown  FlowType = "unknown"
+)
+
+// FlowTypeDetails contain list of valid FlowTypeDetail
+var FlowTypeDetails = []FlowTypeDetail{
```

Comment:
Yes, considered that, but since perf was not an issue I preferred to use a list initially. 

Note, flow type details are not used in critical path, only used when reading configurations.

I changed to map here 84d325b97 

For readability, it indeed improves a bit the  GetFlowTypeByName method.